index.html

```diff
@@ -113,12 +113,12 @@ <h3 class="titles download__title">Zacznij pracować z Alinką!</h3>
       />
     </section>
     <footer class="footer">
-      <div class="icons footer__icons">
-        <img
-          src="assets/images/Code_for_Poznan_final-01.png"
-          alt="Code for Poznan logo"
-          class="icons__logo"
-        />
+            <img
+            src="assets/images/Code_for_Poznan_final-01.png"
+            alt="Code for Poznan logo"
+            class="icons__logo"
```

Comment:
something here is wrong, when you are using BEM, first you should have block, in this case, `icons` then you can have elements of this block like `logo`, so version before with `div class="icons'` was ok :)

Reply:
for me logo is not an icon so I've changed it into footer__logo